src/Graph.spec.ts

```diff
@@ -10,14 +10,30 @@ const INITAL_BOARD: Board = [
 ];
 
 describe("boardToGraph", () => {
-  it("should return a well formed Graph when a Board is passed as argument with the boardToGraph method", () => {
-    const GRAPH: Graph = boardToGraph(INITAL_BOARD);
-
-    expect(GRAPH.nodes).toBeTruthy();
-    expect(GRAPH.nodes["0,0"].value).toBe(1);
-    expect(GRAPH.nodes["1,1"].isExit).toBe(false);
-    expect(GRAPH.nodes["0,1"].isExit).toBe(true);
-    expect(GRAPH.edges[0].from).toBe("0,0");
-    expect(GRAPH);
+  it("should return a well formed Graph when a Board is passed as argument with the boardToGraph function", () => {
+    const graph: Graph = boardToGraph(INITAL_BOARD);
+
+    expect(graph.nodes).toBeTruthy();
+    expect(graph.nodes["0,0"].value).toBe(1);
+    expect(graph.nodes["1,1"].isExit).toBe(false);
+    expect(graph.nodes["0,1"].isExit).toBe(true);
+    expect(graph.edges[0].from).toBe("0,0");
+    expect(graph);
   });
 });
+
+describe("graphToBoard", () => {
+  it("should return a well formed Board when a graph is passed as argument with the graphToBoard function", () => {});
```

Comment:
The it function is not correctly written

Reply:
You can add some eslint rules with jest to avoid this mistakes, see https://www.npmjs.com/package/eslint-plugin-jest

Reply:
Corrected